### DISCUSSION TITLE
Tailwind with React Bazel Project

### DISCUSSION THREAD

**Discussion by amadasunm1** [tailwindlabs/tailwindcss#8051] [Help]
Hello, I have been trying to set up tailwindcss with Bazel for a while and need a bit of help to understand what I may be doing wrong. 

I have taken the following steps:

1. Install tailwindcss, postcss, and autoprefixer as packages
2. Followed the steps in https://github.com/bazelbuild/rules_postcss to update my WORKSPACE
3. Add the following to my BUILD.bazel:
```
postcss_plugin(
    name = "autoprefixer",
    node_require = "autoprefixer",
    deps = ["@npm//autoprefixer"],
)

postcss_plugin(
    name = "tailwindcss",
    node_require = "tailwindcss",
    deps = ["@npm//tailwindcss"],
)

postcss_binary(
    name = "style_processed",
    src = "dev.css",
    named_data = {"tailwindConfig": "tailwind.config.js"},
    plugins = {
        ":tailwindcss": "[{config: bazel.data.tailwindConfig[0]}]",
        ":autoprefixer": "{}",
    },
)
```
4. Run bazel build.

When I run the build I can see the generated binaries in bazel-out but I am not sure what to do from here. Does Tailwind Integrate with Bazel? I've seen the doc on [installation with postcss](https://tailwindcss.com/docs/installation/using-postcss), but have not had any luck so far. 

Any help will be greatly appreciated.

**Comment by Ishaq1189** [comment]
Let us know if you have been able to successfully use tailwind together with Bazel

**Comment by pselamy** [comment]
I am preparing to give these resources a try:

https://github.com/bazelbuild/rules_nodejs/issues/2573
https://github.com/flolu/angular-bazel-architect/commit/1bdf950f66a5cde634495481d4fd065e3a68e64f

I'll let you know how it works out for me.
